dandiapi/api/apps.py

```diff
@@ -50,13 +50,15 @@ def ready(self):
         import dandiapi.api.checks  # noqa: F401, RUF100
         import dandiapi.api.signals  # noqa: F401
 
+        logging.getLogger(__name__).setLevel(settings.DANDI_LOG_LEVEL)
```

Comment:
This was originally `logging.getLogger(__name__.split('.')[0])`, but looking through the codebase and at the logging docs, it seems `logging.getLogger(__name__)` is a standard practice. Was there a reason you had it the other way? (If necessary, we can revert 6addfacce0d6fac0bf11c0da77e038be412ce344, which makes the change.)

Reply:
Actually, this does not have the desired effect (of changing the logging level for all loggers). Instead, we need to omit the `__name__` argument so as to affect the root logger. See b07bfe126db504ab7c879b37eefbad647c2baac3.